.github/workflows/ci.yml

```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - master
 
+env:
+  SYMFONY_PHPUNIT_VERSION: '9.3'
```

Comment:
the phpunit-bridge should take care of this. I'm not comfortable with maintaining a hardcoded version in this repository :-/

Reply:
Okey. That is fair. 

Im not happy with rewriting config in CI =)

So, the question is: What warning do we like?
Option one: Deprecation warning (shown in the description)
Option two: A unknown config warning: 

<img width="689" alt="Screenshot 2020-08-20 at 14 50 03" src="https://user-images.githubusercontent.com/1275206/90772141-889b5400-e2f4-11ea-9447-8fba17146fc4.png">


Im fine with whatever. When I started the PR I did not know of the "unknown config warning".

Reply:
hmm "unknown config" is just a warning or a behavior is not enabled?
I guess it's about coverage, and given we don't use it, it's not a real issue.

I'm fine with beeing compatible with the last version (fix deprecation). But the lower version should work as expected.

Reply:
I agree, it is no real issue. 

I've updated the PR